index.bs

```diff
@@ -3248,6 +3248,7 @@ To <dfn>process an attribution source</dfn> given an [=attribution source=] |sou
     : [=randomized response output configuration/trigger specs=]
     :: |source|'s [=trigger specs=]
 
+1. Let |states| be the [=obtain a set of possible trigger states|number of possible trigger states=] obtained with |randomizedResponseConfig|.
```

Comment:
Optionally, we can pass |states| to channel capacity computation and randomized response algorithm to avoid duplicate computation on the number of possible trigger states. Otherwise, this should be moved close to where it is used.

Reply:
Yes, These changes are covered in https://github.com/WICG/attribution-reporting-api/pull/1364/files

We can resolve the merge conflicts after one the PRs is submitted.

Reply:
Ok, for now, let's just move it down to where it's used.